Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -65,8 +65,8 @@ def check_xcode_up_to_date
         return unless MacOS::Xcode.installed? && MacOS::Xcode.outdated?
 
         message = <<-EOS.undent
-          Your Xcode (#{MacOS::Xcode.version}) is outdated
-          Please update to Xcode #{MacOS::Xcode.latest_version}.
+          Your Xcode (#{MacOS::Xcode.version}) is outdated.
+          Please update to Xcode #{MacOS::Xcode.latest_version} (or delete it).
```

Comment:
I already know I need Xcode (for MacVim).  I don't know whether, on top of this, I should have the CLT.